COOKBOOK-FIXERS.md

```diff
@@ -1,13 +1,13 @@
-Cookbook - Making a new Fixer for PHP CS Fixer
+Cookbook - Making a new Fixer for PHP-CS-Fixer
```

Comment:
:-1:

http://fabien.potencier.org/php-cs-fixer-finally-reaches-version-1-0.html
name should not contain `-`, only file should